auditlog/models/http_request.py

```diff
@@ -47,20 +45,14 @@ def current_http_request(self):
         first call.
         If no HTTP request is available, returns `False`.
         """
+
         if not request:
             return False
         http_session_model = self.env['auditlog.http.session']
         httprequest = request.httprequest
         if httprequest:
             if hasattr(httprequest, 'auditlog_http_request_id'):
-                # Verify existence. Could have been rolled back after a
-                # concurrency error
-                self.env.cr.execute(
-                    "SELECT id FROM %s WHERE id = %s", (
-                        AsIs(self._table),
-                        httprequest.auditlog_http_request_id))
-                if self.env.cr.fetchone():
-                    return httprequest.auditlog_http_request_id
+                return httprequest.auditlog_http_request_id
```

Comment:
This piece of code was added maybe in 8.0 or 9.0 to avoid some error, are you sure it is not required anymore? You got some problem with it?